data/fleet variants.txt

```diff
@@ -0,0 +1,340 @@
+	"Fury" 2
+	"Fury (Missile)"
+	"Fury (Bomber)"
+	"Fury (Laser)"
+
+variant "Fury (Miner)"
+	"Fury"
+	"Fury (Miner)" 2
+
+variant "Fury (Pirate)"
+	"Fury" 2
+	"Fury (Missile)"
+	"Fury (Bomber)"
+	"Fury (Laser)"
+	"Fury (Gatling)"
+	"Fury (Heavy)"
+
+variant "Fury (Pirate North)"
+	"Fury" 2
+	"Fury (Missile)"
+	"Fury (Laser)"
+	"Fury (Bomber)"
+	"Fury (Heavy)"
+
+variant "Gunboat"
+	"Gunboat"
+
+variant "Hauler (Hai)"
+	"Hauler"
+	"Hauler (Hai)"
+
+variant "Hauler II (Hai)"
+	"Hauler II"
+	"Hauler II (Hai)"
+
+variant "Hauler III (Hai)"
+	"Hauler III"
+	"Hauler III (Hai)"
+
+variant "Hawk (Militia)"
+	"Hawk" 2
+	"Hawk (Rocket)"
+	"Hawk (Plasma)"
+	"Hawk (Speedy)"
+	"Hawk (Bomber)"
+
+variant "Hawk (Miner)"
+	"Hawk"
+	"Hawk (Miner)" 2
+
+variant "Hawk (Pirate)"
+	"Hawk"
+	"Hawk (Speedy)"
+	"Hawk (Rocket)"
+
+variant "Hawk (Pirate North)"
+	"Hawk"
+	"Hawk (Speedy)"
+	"Hawk (Rocket)"
+	"Hawk (Bomber)"
+
+variant "Headhunter"
+	"Headhunter" 2
+	"Headhunter (Particle)"
+
+variant "Headhunter (Deep)"
+	"Headhunter" 3
+	"Headhunter (Particle)"
+	"Headhunter (Strike)"
+
+variant "Headhunter (Miner)"
+	"Headhunter"
+	"Headhunter (Miner)" 2
+
+variant "Headhunter (Hai)"
+	variant "Headhunter"
+	"Headhunter (Hai)"
+
+variant "Heavy Shuttle"
+	"Heavy Shuttle" 2
+	"Heavy Shuttle (Armed)"
+
+variant "Lance"
+	"Lance"
+	"Lance (Gatling)"
+
+variant "Leviathan"
+	"Leviathan"
+	"Leviathan (Heavy)"
+	"Leviathan (Laser)"
+
+variant "Leviathan (Hai)"
+	variant "Leviathan" 2
+	"Leviathan (Hai Weapons)"
+	"Leviathan (Hai Engines)"
+
+variant "Manta"
+	"Manta"
+	"Manta (Proton)"
+
+variant "Modified Argosy"
+	"Modified Argosy"
+	"Modified Argosy (Heavy)"
+	"Modified Argosy (Blaster)"
+	"Modified Argosy (Missile)"
+
+variant "Mule"
+	"Mule"
+	"Mule (Heavy)"
+
+variant "Mule (Hai)"
+	variant "Mule" 2
+	"Mule (Hai Engines)"
+	"Mule (Hai Weapons)"
+
+variant "Nest"
+	"Nest"
+	"Nest (Sidewinder)"
+
+variant "Osprey"
+	"Osprey" 2
+	"Osprey (Missile)"
+	"Osprey (Laser)"
+
+variant "Protector"
+	"Protector"
+	"Protector (Laser)"
+	"Protector (Proton)"
+
+variant "Quicksilver"
+	"Quicksilver"
+	"Quicksilver (Proton)"
+
+variant "Rainmaker"
+	"Rainmaker"
+
+variant "Raven"
+	"Raven" 2
+	"Raven (Heavy)"
+	"Raven (Afterburner)"
+
+variant "Roost"
+	"Roost"
+	"Roost (Sidewinder)"
+
+variant "Sparrow (Pirate)"
+	"Sparrow" 2
+	"Sparrow (Gatling)"
+
+variant "Splinter"
+	"Splinter" 2
+	"Splinter (Laser)"
+	"Splinter (Proton)"
+
+variant "Star Barge"
+	"Star Barge" 3
+	"Star Barge (Armed)" 2
+
+variant "Star Queen (Hai)"
+	"Star Queen"
+	"Star Queen (Hai)"
+
+variant "Vanguard"
+	"Vanguard" 2
+	"Vanguard (Particle)"
+	"Vanguard (Missile)"
+
+variant "Wasp"
+	"Wasp" 2
+	"Wasp (Proton)"
+	"Wasp (Meteor)"
+
+variant "Wasp (Pirate)"
+	"Wasp (Gatling)"
+	"Wasp (Javelin)"
+	"Wasp (Meteor)"
+	"Wasp (Plasma)"
+	"Wasp (Proton)"
+
+# Multi-ship variants
+
+variant "Core Pirate Interceptors"
+	variant "Wasp (Pirate)" 5
+	variant "Hawk (Pirate)"
+	variant "Sparrow (Pirate)"
```

Comment:
These multi-ship variants were more so made as examples of what can be done, but should I go through and create such variants wherever possible?

One thing to note about variants like these is that they're probably best used by pulling a single ship from them, at most two (as a stretch), but not more. Reason being that if I were for to example create a "Northern Pirate Medium Warships" variant containing the Firebird, Mule, and Corvette variants, then use that multi-ship variant wherever possible, you'd certainly see a higher variety in what type of fleets you see, but that'd come at the cost of a loss of perceived structure. Right now ships of the same model will often appear in groups (e.g. two or three of the same ship), but if you were to instead do `variant "Northern Pirate Medium Warships" 3`, you'd see any mix of those three ships, which outside of them all being medium warships could look rather unorganized, as if the pirates are just throwing together whatever they can. Now that perception might be exactly what we want from pirates, but such multi-fleet variants likely won't be desirable for organized factions like the FW/Militia, Syndicate, Navy, and perhaps even merchants. (Some multi-fleet variants could be used in some merchant fleet variants, making it look like some merchants are just using whatever they can get their hands on, but on the whole it wouldn't be a good appearance for merchants.)

As I say that, though, I've used the "Human Fighters" mutli-ship variant in greater quantities. Since it consists of only two ships, this doesn't lead to as much of a disorganized looking outcome.

Reply:
Good considerations. For me it is fine to just leave the data-file changes here as-is and do further data-file changes in follow-up PRs if you feel that such updates are beneficial.